controllers/cloud.redhat.com/makers/maker.go

```diff
@@ -245,6 +245,18 @@ func initJob(app *crd.ClowdApp, env *crd.ClowdEnvironment, cj *batch.CronJob, nn
 	cj.Spec.JobTemplate.ObjectMeta.Labels = labels
 	cj.Spec.JobTemplate.Spec.Template.ObjectMeta.Labels = labels
 
+	if job.ConcurrencyPolicy == "" {
+		cj.Spec.ConcurrencyPolicy = batch.ForbidConcurrent
+	} else {
+		cj.Spec.ConcurrencyPolicy = job.ConcurrencyPolicy
+	}
+
+	if job.StartingDeadlineSeconds == nil {
+		cj.Spec.StartingDeadlineSeconds = utils.Int64Ptr(0)
+	} else {
+		cj.Spec.StartingDeadlineSeconds = job.StartingDeadlineSeconds
```

Comment:
Should this be the default? If != nil, add it. Else leave it out of the spec and take the default? https://v1-16.docs.kubernetes.io/docs/tasks/job/automated-tasks-with-cron-jobs/#starting-deadline